components/apimgt/org.wso2.carbon.apimgt.rest.api.common/src/main/java/org/wso2/carbon/apimgt/rest/api/common/RestApiCommonUtil.java

```diff
@@ -397,6 +392,47 @@ public static void validateUserTenantWithAPIIdentifier(APIIdentifier apiIdentifi
         }
     }
 
+    /**
+     * Getting the Identifier of an Organization
+     * @param organizationId UUID of the Organization which the API belongs to
+     * @param tenantDomain   TenantDomain which the API belongs to
+     * @return tenantDomain or OrganizationUUID
+     */
+    public static String getOrgIdMatchesTenantDomain(String organizationId, String tenantDomain)
```

Comment:
getMatchingOrgIdOrTenantDomain?

Reply:
In this method, we are setting the **orgId** as organizationId or tenantDomain. And that tenantDomain is a user given (optional) parameter.

If this method does not signify the logic shall we rename that to one of the following:

1. getOrgId
2. getOrgIdFromTenantDomain
3. retriveOrgIdFromOrganizationIdentifiers

Please suggest a suitable name for this :)